js/script.js

```diff
@@ -0,0 +1,56 @@
+//  Adding books to library
+const booksDisplay = document.querySelector('#books-display');
+const titleInput = document.querySelector('#book-title');
+const authorInput = document.querySelector('#author-name');
+const addBook = document.querySelector('#add-btn');
+addBook.addEventListener('click', () => {
+  if (titleInput.value === '' && authorInput.value === '') {
+    alert('Kindly fill in the inputs');
+  } else {
+    const books = document.createElement('div');
+    const newTitle = document.createElement('h');
+    newTitle.innerHTML = titleInput.value;
+    books.appendChild(newTitle);
+    const newAuthor = document.createElement('p');
+    newAuthor.innerHTML = authorInput.value;
+    books.appendChild(newAuthor);
+    booksDisplay.appendChild(books);
+
+    const removeBtn = document.createElement('button');
+    removeBtn.innerHTML = 'Remove Book';
+    books.appendChild(removeBtn);
+    removeBtn.addEventListener('click', () => {
+      booksDisplay.removeChild(books);
+    });
+  }
```

Comment:
- [ ] Your book removal does not work as expected. All changes should be saved in the local storage. When I removed all the books from the page, the one in the localStorage was still there. Kindly fix that.